st2common/st2common/util/loader.py

```diff
@@ -189,9 +192,10 @@ def register_query_module(module_name):
 
     LOG.debug('Loading query module from: %s', module_path)
 
-    module = imp.load_source(module_name, module_path)
+    if module_name not in QUERIER_MODULES:
+        QUERIER_MODULES[module_name] = imp.load_source(module_name, module_path)
```

Comment:
Let's please add an `else` and LOG.info / debug statement which says that module x for runner y is already cached - this way user won't be that surprised if they made a change without restarting action runner and wondering why the change is not taking effect.